### PR TITLE
[git] [checkout] Include Remote Branches

### PR DESCRIPTION
**Pull request by Monkeyank**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Enhancement
**What is the current behavior? (You can also link to an open issue here)**
#369 Only local branches are available to checkout
**What is the new behavior (if this is a feature change)?**
Adds remote branches to the generator
**Additional info:**

### CODE REVIEW

File: dev/git.ts [hunk 1]
```diff
@@ -45,6 +45,38 @@ const postProcessTrackedFiles: Fig.Generator["postProcess"] = (
...
+    const parts = elm.match(/\S+/g);
+    if (parts.length > 1) {
+      if (parts[0] == "*") {
+        // Current branch.
+        return {
+          name: elm.replace("*", "").trim(),
+          description: "current branch",
+          icon: "⭐️",
+        };
+      } else if (parts[0] == "+") {
```
Comment (fedeci) [inline dev/git.ts:66]: Not relevant in this situation but strict equality (`===`) is generally better.

**Review by fedeci — APPROVED** [review]

**Comment by QuiiBz** [comment]
I'm unable to get this working - could you try and tell me if it's working for you?

**Comment by fedeci** [comment]
I'll try it later.

**Comment by fedeci** [comment]
I confirm it works for me.

**Comment by QuiiBz** [comment]
Got it working, sorry - it was an issue from my side. Let's merge this!

**Review by QuiiBz — APPROVED** [review]
LGTM!

**Pull request merged**